test/integration/controlplane/transformation/transformation_test.go

```diff
@@ -87,27 +87,35 @@ type transformTest struct {
 	secret            *corev1.Secret
 }
 
-func newTransformTest(tb testing.TB, transformerConfigYAML string, reload bool, configDir string, storageConfig *storagebackend.Config) (*transformTest, error) {
+type transformTestConfig struct {
+	transformerConfigYAML string
+	reload                bool
+	configDir             string
+	storageConfig         *storagebackend.Config
+	runtimeConfig         []string
+}
```

Comment:
The number of parameters for `newTransformTest` was already large, with several of them often just the null value, therefore I switched to this struct when adding `runtimeConfig`.